docs/guides/deployment/deploy-to-fly-io-with-sqlite.md

```diff
@@ -1,54 +1,48 @@
 # Deploy Platformatic Applications to Fly.io
 
-## Deploying a Platformatic Runtime Application 
+## Deploying a Platformatic Watt Application 
 
-This guide provides instructions on deploying a Platformatic Runtime application to Fly.io. With a runtime application, you are deploying your entire application, including all services in the `services` folder.
+This guide provides instructions on deploying a Platformatic Watt application to Fly.io. With a Watt application, you are deploying your entire application, including all services in the `web` folder.
 
+### Dockerfile for JavaScript Watt Application
 
-### Dockerfile for Runtime Application
-
-Here is an example Dockerfile for a Platformatic Runtime application:
+Here is an example Dockerfile for a Platformatic JavaScript Watt application:
 
 ```dockerfile
-FROM node:20-alpine AS builder
-
-ENV APP_HOME=/home/app/node/
-WORKDIR $APP_HOME
-
-COPY package.json package-lock.json ./
-COPY services/devotion/package.json services/devotion/package.json
-
-RUN npm ci 
-
-COPY . .
-
-RUN npx platformatic compile
-
-FROM node:20-alpine
-
-ENV APP_HOME=/home/app/node/
-WORKDIR $APP_HOME
-
-COPY package.json package-lock.json ./
-RUN npm ci --only=production
-
-COPY --from=builder $APP_HOME/dist ./dist
-
+ARG NODE_VERSION=20
+FROM node:${NODE_VERSION}-alpine 
+WORKDIR /app
+COPY package.json .
```

Comment:
I think we should copy `package-lock-json` as well

Reply:
I don't think as it installs the `package-lock.json` after npm installation